docs/src/apimanual.md

```diff
@@ -290,9 +306,8 @@ add_constraint(model, VectorOfVariables([x,y,z]), SecondOrderCone(3))
 
 Below is a list of common constraint types and how they are represented
 as function-set pairs in MOI. In the notation below, ``x`` is a vector of decision variables,
-``x_i`` is a scalar decision variable, and all other terms are fixed constants.
-
-[Define notation more precisely. ``a`` vector; ``A`` matrix; don't reuse ``u,l,b`` as scalar and vector]
+``x_i`` is a scalar decision variable, ``\alpha, \beta`` are scalar constants,
+``a, b`` are a constant vectors and `A` is a constant matrix.
```

Comment:
There's another TODO in the section below that I missed in #780: [Define ``\mathbb{R}_+, \mathbb{R}_-``]. You don't have to address it in this PR, but we should leave that issue open if not.